resources/terraform/aws-eks/variables.tf

```
@@ -0,0 +1,17 @@
+variable "region" {
+  type        = string
+  description = "The region in which the cluster is hosted."
+}
+
+variable "kubernetes_version" {
+  type        = string
+  description = "The Kubernetes version."
+  default     = "1.22"
```

```suggestion
  default     = "1.23"
```

Bump to the [latest supported version](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)? It's not a huge deal but, as AWS removes support for older versions, eventually `1.22` will no longer be valid. The same is true for `1.23` but at least that buys us more runway before we have to update this value.